src/components/ui-shell/_shell-switcher.scss

```diff
@@ -0,0 +1,160 @@
+@import '../../globals/scss/functions';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/typography';
+@import 'theme';
+@import 'functions';
+
+@mixin shell-switcher {
+  .#{$prefix}--panel--overlay {
+    position: fixed;
+    top: units(6);
+    left: unset;
+    right: 0;
+    bottom: 0;
+    width: units(40);
+    will-change: transform;
+    transform: translate3d(100%, 0, 0);
+    padding: 1rem 0;
+    overflow-y: auto;
+    z-index: 1000;
+    background-color: $shell-header-bg-02;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+    height: 100%;
+    overflow-x: hidden;
+  }
+
+  .#{$prefix}--panel--expanded {
+    transform: translate3d(0, 0, 0);
+    transition: transform 0.11s cubic-bezier(0.2, 0, 0.38, 0.9);
+  }
+
+  .#{$prefix}--switcher__search-container {
+    padding: 0 units(2);
+    margin-bottom: units(3);
+  }
+
+  .#{$prefix}--search--shell input {
+    background-color: #fff;
+  }
+
+  .#{$prefix}--switcher__item {
+    padding: 3px;
```

Comment:
Ideally these should be in 4, 8, 16, etc. Is the 3px from the comp?

Reply:
This was because the outline on focus is 3px and it was only showing the outline on the top and bottom and not on the sides. should i remove this instead?